package/config.ini

```diff
@@ -6,13 +6,20 @@
 name = preview
 ip = 127.78.0.6
 port = 10000
-timeout = 30
+timeout_in_seconds = 30
 
 image_name = image
 health_name = health
 pdf_name = pdf
 document_name = document
 
+# when this option is set to true it will enable the docs-core dependency
```

Comment:
I would apply this comment for both configs changing all in plural (eg. `when these options are...`)